test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/ContractMintHTSSuite.java

```diff
@@ -357,9 +357,9 @@ private HapiApiSpec transferNftAfterNestedMint() {
 														getNestedContractAddress(innerContract, spec),
 														asAddress(spec.registry().getTokenID(nonFungibleToken)))
 														.bytecode(outerContract)
-														.gas(100_000),
-												newKeyNamed(DELEGATE_CONTRACT_KEY_NAME).shape(revisedKey.signedWith(sigs(ON,
-														outerContract, innerContract))),
+														.gas(300_000),
+												newKeyNamed(DELEGATE_CONTRACT_KEY_NAME).shape(DELEGATE_CONTRACT_KEY_SHAPE.signedWith(sigs(ON,
```

Comment:
We need tests where only the delegate key or contract key are the keys on the token.  I think this and the other contracts are 1 of 2 with and ed25519 and a contract key.  We need End to End tests where it is only 1 of 1 on the contract key, to match customer expectations of a "locked token".